src/middlewares/token.ts

```diff
@@ -0,0 +1,15 @@
+import jwt from "jsonwebtoken"
+import { Request, Response, NextFunction } from "express";
+import { constants } from "../config/constants";
+export default (req: Request, res: Response, next: NextFunction) => {
+    const token = req.get("Authorization");
+    if(!token) {
+        return res.sendStatus(401);
+    }
+    jwt.verify(token, constants.JWTkey, (err: any, decoded: object) => {
+        if (err) {
+            return res.sendStatus(403);
+        }
+        next();
```

Comment:
siempre deja un espacio antes y despues de un bloque IF, FOR, MAP, FIND, FILTER, REDUCE, etc